### PR TITLE
Adds some Irish English localisations

### PR DESCRIPTION
**Pull request by mgriffin**
I've added some Irish specific information, specifically addresses, phone numbers, names and internet information.

### CODE REVIEW

**Comment by fzaninotto** [comment]
Please update in the next week, or I'll close the PR.

File: src/Faker/Provider/en_IE/Internet.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+{
+    protected static $freeEmailDomain = array('gmail.com', 'yahoo.com', 'hotmail.com', 'eircom.net', 'oceanfree.net');
+    protected static $tld = array('ie', 'ie', 'ie', 'ie', 'com', 'com', 'net');
+
+    /**
+     * Converts Irish characters to their ASCII representation
+     *
+     * @return string
+     */
+    public static function toAscii($string)
```
Comment (fzaninotto) [inline src/Faker/Provider/en_IE/Internet.php]: Not required anymore, transliteration is built-in the default provider.

File: src/Faker/Provider/en_IE/Person.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        '{{firstNameMale}} {{lastNameMale}}',
+        '{{firstNameFemale}} {{lastNameFemale}}',
+        '{{firstNameMale}} {{lastNameMale}}',
+        '{{firstNameFemale}} {{lastNameFemale}}',
+        '{{firstNameMale}} {{lastNameMale}}',
+        '{{firstNameFemale}} {{lastNameFemale}}',
+    );
+
+    protected static $firstNameMale = array(
+        'Mícheál', 'Seán', 'Oisín', 'Fionn'
```
Comment (fzaninotto) [inline src/Faker/Provider/en_IE/Person.php]: Not enough data to make a real provider...

**Comment by fzaninotto** [comment]
Tests seem to fail...

**Comment by mgriffin** [comment]
I'm going to close out this PR. I'm not using Faker any more and won't be able to get this section to the required standard.

Sorry for wasting your time on it :(

**Pull request closed without merging**